src/SiteSettings.php

```diff
@@ -95,7 +95,7 @@ protected function getEnv(string $variableName) : ?string {
   public function setDefaults() : void {
     $this->settings['trusted_host_patterns']        = [];
     $this->settings['update_free_access']           = FALSE;
-    $this->configDirectories[CONFIG_SYNC_DIRECTORY] = implode(
```

Comment:
As we are not assigning into `configDirectories[CONFIG_SYNC_DIRECTORY` anymore, are we not using it anywhere else ? If no, Can we remove it ? Also are we using `config_directories` var at all ?

Reply:
configDirectory is used just not the CONFIG_SYNC_DIRECTORY key reference.

Reply:
Yes, I think we can remove this.
It seems like there is no use of it. I checked it on this link. 
https://github.com/wearewondrous/psh-toolbelt/search?q=config_directories
Please review.